filters/include/pcl/filters/functor_filter.h

```diff
@@ -9,6 +9,9 @@
 
 #pragma once
 
+#define PCL_NO_PRECOMPILE
```

Comment:
Errr.... What's this code supposed to do? The include is alive in L15

Moreover, not including the header will cause a compile error, right?

Reply:
Fixed with 98e44f7 as discussed